tasks/responsive_images.js

```diff
@@ -313,11 +313,22 @@ module.exports = function(grunt) {
   var processImage = function(srcPath, dstPath, sizeOptions, tally, callback) {
     var image = gfxEngine(srcPath);
 
-    image.identify(function(err, data) {
+    image.identify("%m:%T:%s\n", function(err, dataRaw) {
       if(err){
         handleImageErrors(err, sizeOptions.engine);
```

Comment:
handleImageErrors calls grunt.fail so the task is aborted anyway. See http://gruntjs.com/api/grunt.fail